internal/kibana/synthetics/schema.go

```diff
@@ -307,6 +334,29 @@ func httpMonitorFieldsSchema() schema.Attribute {
 				Computed:            true,
 				PlanModifiers:       []planmodifier.List{listplanmodifier.UseStateForUnknown()},
 			},
+			"ssl_certificate_authorities": schema.ListAttribute{
+				ElementType:         types.StringType,
+				Optional:            true,
+				MarkdownDescription: "The list of root certificates for verifications is required.",
+			},
+			"ssl_certificate": schema.StringAttribute{
+				Optional:            true,
+				MarkdownDescription: "Certificate.",
+				Computed:            true,
+				PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+			},
+			"ssl_key": schema.StringAttribute{
```

Comment:
Shall we set `Sensitive` for both `ssl_key` and `ssl_key_passphrase` attributes?

Reply:
Good idea, will do